frontend/src/components/MedicalRecord/MedicalRecord.jsx

```
@@ -1,228 +1,228 @@
+            <div className="w-24 h-24 bg-gray-300 rounded-full flex items-center justify-center">
+              <span className="text-xl font-bold text-gray-600">{firstLetter}</span>
+            </div>
+          )}
+          <input
+            type="file"
+            ref={fileInputRef}
+            style={{ display: 'none' }}
+            accept="image/*"
+            onChange={handlePhotoChange}
+          />
+          <button onClick={handleAddPhotoClick} className="mt-3 text-blue-600 hover:text-blue-800">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)